chameleonultragui/lib/main.dart

```diff
@@ -77,16 +80,10 @@ class MyApp extends StatelessWidget {
 class MyAppState extends ChangeNotifier {
   final SharedPreferencesProvider sharedPreferencesProvider;
   MyAppState(this.sharedPreferencesProvider);
-
-  bool onAndroid =
-      Platform.isAndroid; // Are we on android? (mostly for serial port)
-
-// Android uses AndroidSerial, iOS can only use BLESerial
-// The rest (desktops?) can use NativeSerial
-  AbstractSerial connector = Platform.isAndroid
-      ? AndroidSerial()
-      : (Platform.isIOS ? BLESerial() : NativeSerial());
-
+  // State
+  bool onWeb = kIsWeb;
+  bool onAndroid = !kIsWeb && Platform.isAndroid; // Are we on android? (mostly for serial port)
+  AbstractSerial connector = SerialConnector(); // Chameleon Object, connected Chameleon
```

Comment:
Platform.isIOS missing there, thus breaking iOS

Reply:
Not sure I understand, why is it breaking? The switch between Platforms is happening here: https://github.com/GameTec-live/ChameleonUltraGUI/blob/53f5982487a971540bca81d58decf5869cf5c370/chameleonultragui/lib/main.dart#L15?

Reply:
Yea, saw it later, not included in final review